docs/guides/primitives.mdx

```diff
@@ -187,6 +193,12 @@ list2 = [SparsePauliOp("XX"), SparsePauliOp("XY"), SparsePauliOp("IZ")] # shape
 
 #### Sampler V1
 
+<Admonition type="warning">
+The [SamplerV1](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.SamplerV1) has been deprecated in `qiskit-ibm-runtime 0.23`.
+Their support will be removed no earlier than July 15, 2024.
```

Comment:
```suggestion
Their support will be removed no earlier than 15 July 2024.
```

Reply:
I changed the date format, but Jessie said that the date was moved to August 15